### ISSUE TITLE
Update needed to IsDefault

### ISSUE THREAD

**Issue by SteveColegrove** [MicrosoftDocs/office-docs-powershell#3370]
Please consider updating the parameter IsDefault.  It says, "This parameter is reserved for internal Microsoft use" which, in my experience, indicates the param can't really be used by the enduser/customer.  It also seems to indicate the only possible values are none and false but not true.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 420e4f76-b48e-394a-6125-9fa99bc02e57
* Version Independent ID: 0b0c673f-0abc-020f-3674-4c624401cab1
* Content: [Set-RetentionPolicy (exchange)](https://docs.microsoft.com/en-us/powershell/module/exchange/policy-and-compliance-retention/set-retentionpolicy?view=exchange-ps#feedback)
* Content Source: [exchange/exchange-ps/exchange/policy-and-compliance-retention/Set-RetentionPolicy.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/policy-and-compliance-retention/Set-RetentionPolicy.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@SteveColegrove, a parameter that contains the description "This parameter is reserved for internal Microsoft use" is likely available, but isn't supposed to be used (or was replaced by another parameter). If the parameter wasn't available, we'd simply remove it from the topic.

The rest of that description tells you to use the RetentionPolicy parameter on the Set-MailboxPlan cmdlet to set the default retention policy, which I'm pretty sure is accurate.

In other words, even if _IsDefault_ is available to you, don't use it.

Thanks for the feedback.
